docs/install/Fedora.md

```diff
@@ -14,7 +14,7 @@ dnf install atomic
 ##Make
 On Fedora, You need to install required build dependencies
 ```
-yum-builddep atomic
+#yum-builddep atomic
```

Comment:
I complie atomic on centos7 error 
W: 91,12: __init__ method from a non direct base class 'super' is called (non-parent-init-called)
W: 86, 8: __init__ method from base class 'Action' is not called (super-init-not-called)
W:136,11: Except doesn't do anything (pointless-except)
W:166,11: Except doesn't do anything (pointless-except)
ERROR:root:Could not find any typelib for OSTree